signup.html

```diff
@@ -0,0 +1,116 @@
+                        <img src="./imgs/turbotax.png" alt="">
+                        <p>turbotax</p>
+                    </div></a>
+                </div>
+            </div>
+            <!-- The end of icons div -->
+        <div class="form-container">
+                <!--a huge container -> (h2,p)/form/ last(p)-->
+                <div class="center-content">
+                    <!-- div contains h2 and p tags-->
+                    <h2>Create an Intuit account</h2>
+                    <p>One account for everything Intuit, including Mint. <span><a href="#">learn more</a></span></p>
+                </div>
+                <!--end of (h2,p) div-->
+
+                <!--start of the form-->
+                <form>
+                    <label>Email address</label><br>
+                    <input type="text"><br><br>
+                    <label> Phone (recommended)</label><br>
+                    <input type="tel">
+                    <span>
+                      Standard call, messaging or data rates may apply.
+                    </span>
+                    <br><br>
+                    <label>Password</label><br>
+                    <input type="password"><br><br>
+                    <button id="create-account" type="submit"><i class="fas fa-lock"></i>Create Account</button>
+                </form>
+                <!--end of form-->
+
+                <div class="terms-privacy">
+                    <!--start of last (p)-->
+                    <p>By clicking Create account, you agree to our
+                        <span><a href="#">Terms</a></span> and have read and acknowledge our <span><a href="#">Privacy Statement</a></span>.</p>
+
+                </div>
+                <!--end of last (p)-->
+                <div class="recaptcha">
+                    <!-- This div will contain the recaptcha -->
+                    <p><span>Invisible reCAPTCHA by Google <a href="#">Privacy Policy</a></span> and <span><a href="#">Terms of use</a></span>.</p>
+                </div>
+                <!-- The end of recaptcha -->
+            </div>
+                <!--end of the huge container -->
+
+
+        </div>
+
+
+    </main>
+    <!-- The end of the main -->
+
+    <footer>
+        <div class="footer-content">
+            <!-- This div contains the links and the p tag-->
+            <div class="footer-links">
+                <!-- This div will contain the links -->
+                <a href="">Legal</a>
+                <a href="">Privacy</a>
+                <a href="">Security</a>
+            </div>
+            <p> &copy; 2020 Intuit, Inc. All rights reserved. Intuit, QuickBooks, QB, TurboTax, ProConnect and Mint are registered trademarks of Intuit Inc.
+                Terms and conditions, features, support, pricing, and service options subject to change without notice.</p>
+        </div>
+    </footer>
+      <button id="privacy-setting-button">Privacy Settings</button>
```

Comment:
**Comment:** This button doesn't appear on the original page in my screen, but it's cool you added the sticky property for positioning 😎